test/flash_test/src/flash_test/flash_test.c

```diff
@@ -41,69 +41,99 @@ flash_cli_cmd(int argc, char **argv)
     uint32_t sz = 1;
     int sec_cnt;
     int i;
+    int devid;
     int soff;
     char *eptr;
     char tmp_buf[8];
     char pr_str[80];
 
-    hf = hal_bsp_flash_dev(0);
-    if (!hf) {
-        console_printf("No flash device present\n");
+    if (argc > 1 && (!strcmp(argv[1], "?") || !strcmp(argv[1], "help"))) {
+        console_printf("Commands Available\n");
+        console_printf("flash <flash-id> -- dumps sector map \n");
```

Comment:
Should this be `[flash-id]` (i.e., this argument is optional)?

Reply:
Yup, that one is optional, updated, thanks!